ods_ci/tests/Resources/Page/ModelRegistry/ModelRegistry.resource

```diff
@@ -0,0 +1,400 @@
+    END
+
+Disable Model Registry If Needed
+    [Documentation]    If we had to enable the component before the test run, let's also disable it at the end to leave
+    ...    the cluster in the same state we found it in
+    IF    ${DISABLE_COMPONENT}==${True}
+        Set Component State    modelregistry    Removed
+        Run And Verify Command    oc delete namespace ${NAMESPACE_MODEL_REGISTRY} --force
+    END
+
+Model Registry Pre Upgrade Scenario
+    [Documentation]    Creates a Model Registry instance and registers a model/version
+    Prepare Model Registry Test Setup
+    Create Workbench    workbench_title=${WORKBENCH_TITLE}    workbench_description=Registry test
+    ...                 prj_title=${PRJ_TITLE}    image_name=Minimal Python  deployment_size=${NONE}
+    ...                 storage=Persistent   pv_existent=${NONE}
+    ...                 pv_name=${NONE}  pv_description=${NONE}  pv_size=${NONE}
+    Workbench Should Be Listed      workbench_title=${WORKBENCH_TITLE}
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE}
+    ${workbenches}=    Create List    ${WORKBENCH_TITLE}
+    Create S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=${DC_S3_NAME}
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=${AWS_BUCKET}    connected_workbench=${workbenches}
+    Data Connection Should Be Listed    name=${DC_S3_NAME}    type=${DC_S3_TYPE}    connected_workbench=${workbenches}
+    Open Data Science Project Details Page       project_title=${prj_title}    tab_id=workbenches
+    Wait Until Workbench Is Started     workbench_title=${WORKBENCH_TITLE}    timeout=120s
+    ${handle}=    Launch And Access Workbench    workbench_title=${WORKBENCH_TITLE}
+    ...    username=${TEST_USER.USERNAME}     password=${TEST_USER.PASSWORD}
+    ...    auth_type=${TEST_USER.AUTH_TYPE}
+    Upload File In The Workbench     filepath=${SAMPLE_ONNX_MODEL}    workbench_title=${WORKBENCH_TITLE}
+    ...         workbench_namespace=${PRJ_TITLE}
+    Upload File In The Workbench     filepath=${JUPYTER_NOTEBOOK_FILEPATH}    workbench_title=${WORKBENCH_TITLE}
+    ...         workbench_namespace=${PRJ_TITLE}
+    Download Python Client Dependencies    ${MR_PYTHON_CLIENT_FILES}    ${MR_PYTHON_CLIENT_WHL_VERSION}
+    Upload Python Client Files In The Workbench    ${MR_PYTHON_CLIENT_FILES}
+    Upload Certificate To Jupyter Notebook    ${CERTS_DIRECTORY}/domain.crt
+    ${self_managed} =    Is RHODS Self-Managed
+    IF  ${self_managed}    Upload Certificate To Jupyter Notebook    openshift_ca.crt
+    Jupyter Notebook Can Query Model Registry     ${JUPYTER_NOTEBOOK}
+    SeleniumLibrary.Switch Window    ${handle}
+    Stop Workbench    workbench_title=${WORKBENCH_TITLE}
+    SeleniumLibrary.Close All Browsers
+
+Model Registry Post Upgrade Scenario
```

Comment:
why not checking that the service is still working from the workbench as well?

Reply:
Tested manually for now so not strictly needed for 2.15, the whole test needs to be expanded but I want automation in place asap.